test/integration/entry-integration.js

```diff
@@ -58,8 +58,8 @@ describe('Entry Api', () => {
 
     test('Gets entries with content type query param', async () => {
       return environment.getEntries({ content_type: 'cat' }).then((response) => {
-        expect(response.total).eql(3)
-        expect(response.items.map((item) => item.sys.contentType.sys.id)).eql(['cat', 'cat', 'cat'])
+        expect(response.total).eql(4)
+        expect(response.items.every((item) => item.sys.contentType.sys.id === 'cat'))
```

Comment:
That's the place I didn't get in my PR. Why the hell does it becomes 4 calls?

Reply:
This was from yesterday: a new cat was added in the test space. I was going out of my mind 🤯

Reply:
Same! That was the reason I couldn't finish my previous PR...